h2o-genmodel/src/main/java/hex/genmodel/algos/glrm/GlrmMojoModel.java

```diff
@@ -151,7 +174,9 @@ public GlrmMojoModel(String[] columns, String[][] domains, String responseColumn
    * @return
    */
   public double applyBestAlpha(double[] u, double[] x, double[] grad, double[] a, double oldObj, Random random) {
-    double[] bestX = new double[x.length];
+    //double[] bestX = new double[x.length];
+    double[] bestX = _tbest;
```

Comment:
The original code creates new array which `bestX` variable points to. This piece of code together with `Arrays.fill(bestX, 0.0);` will make `bestX` point to the shared `_tbest` array and fill it with zeros. Is this intentional ?

Reply:
Not crucial though ...